// [[Reason: Implicit undefined not fixed by code mod yet]] ////

//// [tests/cases/conformance/types/namedTypes/optionalMethods.ts] ////

===================================================================
--- TSC	declarations
+++ DTE	declarations
@@ -8,14 +8,14 @@
     g?(): number;
 }
 declare function test1(x: Foo): void;
 declare class Bar {
-    d?: number | undefined;
+    d?: number;
     e: number;
     a: number;
     b?: number;
     c?: number | undefined;
-    constructor(d?: number | undefined, e?: number);
+    constructor(d?: number, e?: number);
     f(): number;
     g?(): number;
     h?(): number;
 }
